docs/site/downloads/cldr-46.md

```diff
@@ -278,8 +278,8 @@ A few /common/testData/ files have been replaced:
 
 1. Databases that use collation keys are sensitive to any changes in collation, and will need reindexing.
 This can happen with any CLDR release (especially those for a new version of Unicode), but more characters are affected in this release: see a summary [above](#collation-data-changes).
-2. Two collation variants are to be dropped in a v46.1 release: zh-u-co-gb2312 and zh-u-co-big5han.
-These matched the ordering of two legacy character encodings.
+2. Two collation variants are to be dropped in CLDR 47 release: zh-u-co-gb2312 and zh-u-co-big5han.
```

Comment:
optional: change "in CLDR 47 release" to one of
- ... in the CLDR 47 release
- ... in CLDR 47